tests/OpenTK.Tests.Math/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿// <auto-generated/>
```

Comment:
This shouldn't be in source control.

Reply:
Yes I know, I had it removed in another commit in the fixing paket PR.
Seems I haven't quite got paket happy with this right now either. Missing an xunit reference.

Reply:
Great!! I will start to transfer my code to xUnit.